bindings/experimental/distrdf/python/DistRDF/Profiling/Flamegraph.py

```diff
@@ -0,0 +1,97 @@
+from functools import partial
+import os
+from typing import Callable, Optional
+
+from subprocess import run
+
+from DistRDF.Profiling.ErrorHandling import check_and_raise_errors
+from DistRDF.Profiling.Perf import collect_perf_data
+from DistRDF.Profiling.Base import ProfilingData, Visualization
+
+def Decorator(func:Callable, data_dir:str, perf_options:Optional[dict]):
```

Comment:
Style comment: PEP8 suggests using snake case names for functions. We try to adhere to that for all functions that are called internally in DistRDF. Functions that the users might see have to follow the camel case since that's what's done in ROOT C++ classes